python/GafferDelight/__init__.py

```diff
@@ -38,4 +38,21 @@
 
 from ._GafferDelight import *
 
+# GafferDelight does not need any symbols from IECoreDelight, so MSVC tries
+# to be helpful and not load IECoreDelight.dll. Skipping that means that
+# 3Delight is never registered as a renderer, so we import / register it manually.
+
+import os
+
+ieCoreDelight = None
+
+if os.name == "nt" and ieCoreDelight is None :
+
+	import ctypes
+
+	try :
+		ieCoreDelight = ctypes.CDLL( "IECoreDelight.dll" )
+	except :
+		raise ImportError
```

Comment:
In https://github.com/GafferHQ/gaffer/discussions/5040, you mentioned that the `IECoreDelight` module had this same problem, so I think perhaps it would be best to put the fix in `IECoreDelight/__init__.py` instead? I presume that would be necessary to get the IECoreDelightTest module passing. Then we could just `__import__( IECoreDelight )` at the top of _this_ file, so the same fix works for `GafferDelight`.

Reply:
Yes, that's definitely a better solution. Fixed in https://github.com/GafferHQ/gaffer/pull/5082/commits/e95e2f39e5cb206f9514aebc172537c1240b5817